homework2/IStack.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Homework2;
+
+public interface IStack
+{
+    /// <summary>
+    /// Inserts the number to the top of the stack.
+    /// </summary>
+    void Push(double number);
+    /// <summary>
```

Comment:
```suggestion
    void Push(double number);
    
    /// <summary>
```
Методы разделяются пустой строкой всегда